guideline-16.md

```diff
@@ -1,3 +1,3 @@
 <h4>16. A complete plugin must be available at the time of submission.</h4>
 
-All plugins are examined prior to approval, which is why a zip file is required. Names cannot be “reserved” for future use or to protect brands (see #17: respect brands). Directory names for approved plugins that are not used may be given to other developers.
\ No newline at end of file
+All plugins are examined prior to approval, which is why a zip file is required. Names cannot be “reserved” for future use or to protect brands. Directory names for approved plugins that are not used may be given to other developers.
```

Comment:
Is there a period of time that “not used” should mention, to avoid some hypothetical situation where two people want to claim the same directory name?

For example: “Directory name ownership lasts a minimum of 30 days, and ownership can not be changed until that 30 days is up.”

Reply:
There really isn't a period of time. It's a very rare occurrence now that we hard-coded a block in submissions (you can't submit anything new if you have a plugin approved but unused). I don't know that we'd gain all that much by codifying a hard line there.